core/src/test/java/greencity/controller/ManagementOrderControllerTest.java

```diff
@@ -455,4 +455,10 @@ void addPaymentDiscountTest() throws Exception {
             .andExpect(status().isCreated());
 
     }
+
+    @Test
+    void getOrderCancellationReason() throws Exception {
+        this.mockMvc.perform(get(ubsLink + "/get-order-cancellation-reason" + "/{id}", 1L))
```

Comment:
please add `verify` that` ubsManagementService.getOrderCancellationReason` is called with expected params